source/connect.txt

```diff
@@ -101,14 +101,32 @@ Use Advanced Connection Options
 
 .. include:: /includes/intro-advanced-connection-options.rst
 
-For details, see :ref:`advanced-connection-options`.
+For details on how to specify connection options using Compass, 
+see :ref:`advanced-connection-options`.
 
 Connect to Compass from the Command Line
 ----------------------------------------
 
 .. include:: /includes/intro-command-line-connection.rst
 
-For details, see :ref:`comp-command-line-connect`.
+To connect to Compass using the command line, you can include your connection 
+options directly in your connection string or as command line arguments.
+
+To specify connection options in your connection string, use the following format: 
+
+.. code-block::
+
+   mongodb-compass "mongodb://username:password@hostname:port/database?ssl=true&replicaSet=myReplicaSet"
```

Comment:
I'd suggest using single quotes here, since in most UNIX shells that will be the best way to pass a "literal" string, and we have had people come to us with bug reports because they were using double quotes and e.g. `authSource=$external` or something similar that the shell interpreted as special syntax